tools/db_crashtest.py

```diff
@@ -453,8 +453,7 @@ def whitebox_crash_main(args, unknown_args):
 
         stdoutdata = stdoutdata.lower()
         errorcount = (stdoutdata.count('error') -
-                      stdoutdata.count('got errors 0 times') -
-                      stdoutdata.count('got expected errors 0 times'))
```

Comment:
I think the problem might actually be this line: if the errors are expected, we shouldn't expect the number to be zero, no? That is, we might consider keeping the message in `db_stress_stat.h` and simply remove the `0 times` part from here so that any number of expected errors are ignored.

Reply:
I thought of doing that initially, but I don't think that print tells us anything useful anyway.